Tests/PHPUnitTestCase.php

```diff
@@ -0,0 +1,5 @@
+<?php
```

Comment:
It required for test pass on `2.8.0` when composer run with `--prefer-lowest`. Symfony added support of the namespaced PHPUnit from 2.8.18